src/odemis/driver/xt_client.py

```diff
@@ -2235,8 +2235,18 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                     pos["t"] = pos.pop("rx")
                 if "rz" in pos.keys():
                     pos["r"] = pos.pop("rz")
+
                 self.parent.move_stage(pos, rel=rel)
-                time.sleep(0.1)  # It takes a little while before the stage is being reported as moving
+                time.sleep(100e-3)  # It takes a little while before the stage is being reported as moving
+
```

Comment:
Recommend changing to a constant for re-use
```suggestion
                WAIT_DURATION = 100e-3
                time.sleep(WAIT_DURATION)  # It takes a little while before the stage is being reported as moving

```